doc/book/admin/upgrades.rst

```diff
@@ -58,95 +50,78 @@ How to upgrade from Tarantool 1.7 to 2.x
-2. Stop the Tarantool server.
+    Before upgrading Tarantool from 1.x to 2.y, please read about the associated
+    :ref:`caveats <admin-upgrades-1.6-1.10>`.
 
-3. Make a copy of all data (see an appropriate hot backup procedure in
-   :ref:`Backups <admin-backups>`) and the package from which the current (old)
-   version was installed (for rollback purposes).
-
-4. Update the Tarantool server. See installation instructions at Tarantool
-   `download page <http://tarantool.org/download.html>`_.
+Before the upgrade
+~~~~~~~~~~~~~~~~~~
 
-5. Update the Tarantool database. Put the request ``box.schema.upgrade()``
-   inside a :doc:`box.once() </reference/reference_lua/box_once>` function in your Tarantool
-   :ref:`initialization file <index-init_label>`.
-   On startup, this will create new system spaces, update data type names (e.g.
-   num -> unsigned, str -> string) and options in Tarantool system spaces.
+#. Check the replica set health by running the following code on every replica:
 
-6. Update application files, if needed.
+   ..  code-block:: tarantoolsession
 
-7. Launch the updated Tarantool server using ``tarantoolctl`` or ``systemctl``.
+       box.info.ro -- shouldn't be "orphan"
+       box.info.ro_reason -- since 2.10, shouldn't be "orphan"
+       box.info.status -- should be "running"
 
-.. _admin-upgrades_replication_cluster:
+   If ``box.info.ro``, ``box.info.ro_reason``, or ``box.info.status`` have the value ``orphan``,
+   first resolve the replication issues and only then continue.
 
---------------------------------------------------------------------------------
-Upgrading Tarantool in a replication cluster
---------------------------------------------------------------------------------
+   If you're running Cartridge, you can also check node health in the UI.
 
-Tarantool 1.7 can work as a :ref:`replica <replication-architecture>`
-for Tarantool 1.6 and vice versa. Replicas
-perform capability negotiation on handshake, and new 1.7 replication features
-are not used with 1.6 replicas. This allows upgrading clustered configurations.
+#. Make sure each replica connected to the rest of the replica set.
+   Run the following, replacing the ``id`` with the replica ID (the output of ``box.info.id``):
 
-This procedure allows for a rolling upgrade **without downtime** and works for
-any cluster configuration: master-master or master-replica.
+   ..  code-block:: tarantoolsession
 
-1. Upgrade Tarantool at all replicas (or at any master in a master-master
-   cluster). See details in
-   :ref:`Upgrading a Tarantool instance <admin-upgrades_instance>`.
+       box.info.replication[id].upstream
+       box.info.replication[id].downstream
 
-2. Verify installation on the replicas:
+   The ``status`` field in both outputs should have the value ``follow``.
+   This means that the replicas are connected and there are no errors in the data flow.
 
-   a. Start Tarantool.
+   The value of the ``lag`` field should be less than 1 second.
```

Comment:
I think this needs some clarification.
The value of `lag` should be sane, for example, when `box.cfg.replication_timeout` is 1 second, it's generally ok to have lag ~ 10 seconds. I guess only the user knows which values of lag are fine for him.

It's always ok to have lag <= box.cfg.replication_timeout. 
When lag is > box.cfg.replication_timeout the user has to use common sense. A lag of several hours is surely not okay (>= thousands of seconds)

Sorry if I told you otherwise in chat.